livre/eq/Config.cpp

```diff
@@ -60,6 +97,65 @@ class Config::Impl
         , dataFrameRange( INVALID_FRAME_RANGE )
     {}
 
+    void gatherHistogram( const Histogram& histogram, const float area, const uint32_t currentId )
+    {
+        const uint32_t oldestId = histogramQueue.empty() ? -1u :
+                                  histogramQueue.back().id;
+
+        // If we get a very old frame skip it
+        if( oldestId != -1u && currentId < oldestId )
+            return;
```

Comment:
You're not using oldestId for anything after this statement. You can replace the code by:

    if( !histogramQueue.empty() and currentId < histogramQueue.back().id )
        return;

That avoids having to use the spurious value -1u.